JensonChoi/moodify
# Personalize

**changes:** 
added an endpoint to generate personalized playlists which use the user's top 5 artists in the last 6 months as seed artists to generate the new mood playlist.

also added jenson's spotifyapi fix lol

**to test using postman:**
after logging in to Spotify account on localhost:8888, make a `POST` request to `/playlist/personal_generate` with your desired mood in the body of the request (e.g. 'country', 'happy', etc.). **NOTE: need to add support for moods that are not comma-separated**. 
<img width="706" alt="image" src="https://user-images.githubusercontent.com/41043008/222935178-9b768521-8c27-484e-bb84-e5b6de6f008b.png">

send the request - you should receive an array of 20 tracks that are similar to what you already listen to!
<img width="712" alt="image" src="https://user-images.githubusercontent.com/41043008/222935190-656139fd-86e2-4442-b60d-0115a29f4ea0.png">
